src/components/Modals/SetAclPermsDocContainerModal.jsx

```diff
@@ -0,0 +1,172 @@
+// React Imports
+import React, { useContext, useState } from 'react';
+// Custom Hook Imports
+import { useSession, useStatusNotification } from '@hooks';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import Dialog from '@mui/material/Dialog';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import SettingsIcon from '@mui/icons-material/Settings';
+import TextField from '@mui/material/TextField';
+// Utility Imports
+import { runNotification, setDocContainerAclPermission } from '@utils';
+// Context Imports
+import { SignedInUserContext } from '@contexts';
+// Component Imports
+import FormSection from '../Form/FormSection';
+
+/**
+ * @typedef {import("../../typedefs.js").setAclPermsDocContainerModalProps} setAclPermsDocContainerModalProps
+ */
+
+/**
+ * SetAclPermsDocContainerModal Component - Modal component that generates the form for
+ * setting ACL permissions to another user's Documents container in their Solid
+ * Pod via Solid Session
+ *
+ * @memberof Forms
+ * @name SetAclPermsDocContainerModal
+ * @param {setAclPermsDocContainerModalProps} Props - Props for SetAclPermsDocContainerModal component
+ * @returns {React.JSX.Element} The SetAclPermsDocContainerModal Component
+ */
+const SetAclPermsDocContainerModal = ({ showModal, setShowModal }) => {
+  const { session } = useSession();
+  const { state, dispatch } = useStatusNotification();
+  const { podUrl } = useContext(SignedInUserContext);
+  const [permissionState, setPermissionState] = useState({
+    podUrlToSetPermissionsTo: '',
+    permissionType: ''
+  });
+
+  const clearInputFields = () => {
+    dispatch({ type: 'CLEAR_PROCESSING' });
+  };
+
+  const handleCloseModal = () => {
+    setPermissionState({
```

Comment:
Think we can move `setPermissionState` set function to `clearInputFields` instead of `handleCloseModal`.

Reply:
Think this is fine too. Wasn't expecting setShowModal to move from the handler function as well.